app/templates/components/books-deck.hbs

```diff
@@ -41,7 +41,15 @@
         </form>
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+        <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          {{QuantityDefault}}
+        </button>
+        <div class="dropdown-menu" aria-labelledby="Qty">
+          <a class="dropdown-item" href="#"{{action 'addQuantity' 1}}>1</a>
```

Comment:
these quantities should NOT be hard coded. your code should account for how many books there actually are. what if i want to buy all 10??